docs/security/user-certs.md

```diff
@@ -44,74 +44,34 @@ Follow the steps below to get an user certificate:
-error:80066423:lib(128):verify_callback:remote certificate has expired:sslutils.c:1963
-remote certificate has expired
-Function: verify_callback
-error:80066412:lib(128):verify_callback:certificate::sslutils.c:2283
-        subject=/DC=org/DC=DOEGrids/OU=Certificate Authorities/CN=DOEGrids CA 1
-        issuer =/DC=net/DC=ES/O=ESnet/OU=Certificate Authorities/CN=ESnet Root CA 1
-certificate:
-        subject=/DC=org/DC=DOEGrids/OU=Certificate Authorities/CN=DOEGrids CA 1
-        issuer =/DC=net/DC=ES/O=ESnet/OU=Certificate Authorities/CN=ESnet Root CA 1
-Function: verify_callback
-user@host $ voms-proxy-init -debug -voms osg
-Detected Globus version: 2.2
-Unspecified proxy version, settling on Globus version: 2
-Number of bits in key :1024
-Files being used:
- CA certificate file: none
- Trusted certificates directory : /etc/grid-security/certificates
- Proxy certificate file : /tmp/x509up_u20003
- User certificate file: /home/user/.globus/usercred.p12
- User key file: /home/user/.globus/usercred.p12
-Output to /tmp/x509up_u20003
-Enter GRID pass phrase for this identity:
-Your identity: /DC=org/DC=doegrids/OU=People/CN=First Lastname 12345
-Using configuration file /home/marco/.glite/vomses
-Using configuration file /etc/vomses
-Creating temporary proxy to /tmp/tmp_x509up_u20003_17448 ..........................++++++
-...................++++++
- Done
-Contacting  voms.opensciencegrid.org:15027 [/DC=org/DC=doegrids/OU=Services/CN=host/voms.opensciencegrid.org] "osg" Failed
-
-Error: Error during SSL handshake:Either proxy or user certificate are expired.
-
-None of the contacted servers for osg were capable
-of returning a valid AC for the user.
-```
 
-To solve the problem split the bundle in the two PEM files as documented in the [referenced document](http://security.ncsa.illinois.edu/research/grid-howtos/usefulopenssl.html), so that the embedded CA chain is not used.
+### Certificate formats
+
+Your user certficate can be stored in a few different formats.
+The two most common formats used on OSG are the [PKCS12](https://en.wikipedia.org/wiki/PKCS_12) and
+[PEM](https://en.wikipedia.org/wiki/Privacy-Enhanced_Mail) formats.
+In the PEM format, your user certificate is stored in two separate files: one for the certificate and another for the
+private key.
+The PKCS12 format stores the certificate and private key in a single file along with an optional certificate chain.
+Most OSG user tools will work with both but will try to use PEM files first.   
+
+To convert a PKCS12 file to  PEM files, do the following.  
+   
+1. First, extract your user certificate from your PKCS12 file.  We assume that the PKCS12 file is called `usercred.p12` and output the PEM certificate to `usercert.pem`.
+
+        :::console
+        user@host $ openssl pkcs12 -in usercred.p12 -out usercert.pem -nodes -clcerts -nokeys
+        Enter Import Password:
+        MAC verified OK
+   
+1. Second, extract the private key. The private key will be saved to userkey.pem
```

Comment:
Let's mention and differentiate between the two different password prompts

Reply:
done